autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/UpdateFailedHandler.java

```diff
@@ -42,17 +52,12 @@ public void onApplicationEvent(UpdateFailedEvent event) {
                 .orElse(1);
         if (failed < RETRY_THRESHOLD) {
             updateFailures.put(autoscaleClusterId, failed);
-            LOGGER.debug("Increased failed count '{}' for cluster '{}'", failed, cluster.getStackCrn());
+            LOGGER.info("Increased Autoscaling failure count '{}' for cluster '{}'", failed, cluster.getStackCrn());
         } else {
-            suspendCluster(cluster);
+            clusterService.setAutoscaleState(cluster.getId(), false);
             updateFailures.remove(autoscaleClusterId);
-            LOGGER.debug("Suspended cluster monitoring for cluster '{}' due to failing update attempts", cluster.getStackCrn());
-        }
-    }
-
-    private void suspendCluster(Cluster cluster) {
-        if (!cluster.getState().equals(ClusterState.SUSPENDED)) {
-            clusterService.setState(cluster.getId(), ClusterState.SUSPENDED);
+            historyService.createEntry(ScalingStatus.DISABLED, messagesService.getMessage(AUTOSCALING_SUSPENDED), cluster);
+            LOGGER.info("Suspended Autoscaling for cluster '{}' due to repeated failure of scaling attempts", cluster.getStackCrn());
```

Comment:
Would be useful to include the number of failures, and over what duration of time.

Reply:
Time tracking of failures is not implemented in periscope. That would be a new feature.